src/clarity/nav/demo/header-types.demo.html

```diff
@@ -113,6 +113,27 @@
                 </a>
             </div>
         </header>
+
+        <header class="header-6">
+            <div class="branding">
+                <a href="javascript://" class="nav-link">
+                    <clr-icon shape="vm-bug"></clr-icon>
+                    <span class="title">Project Clarity</span>
+                </a>
+            </div>
+            <div class="header-actions">
+                <clr-dropdown class="dropdown bottom-right">
+                    <button class="nav-text" clrDropdownToggle>
+                        john.doe@vmware.com
+                        <clr-icon shape="caret" class="icon-orient-down"></clr-icon>
```

Comment:
This class `icon-orient-down` will be deprecated. Can you use `shape="caret down"`?